### ISSUE TITLE
What is the meaning of parameter filter_t in get_data_CMAPSS.py ?

### ISSUE THREAD

**Issue by LiangHaoran** [nicolasoyharcabal/ConvRNN_for_RUL_estimation#1]
Hi, what is the meaning of parameter filter_t in get_data_CMAPSS.py ?

**Comment by kkangshen** [comment]
我的理解，这里的filter_t是一个滑动窗口，每次取窗口内的数据。

**Comment by kkangshen** [comment]
Run the code and find that the RMSE and Score values of the test set are very large. Is the experimental result of the paper the result of the train set and the validation set?

**Comment by hutchenky** [comment]
> 运行代码，发现测试集的RMSE和得分值非常大。论文的实验结果是训练集和验证集的结果吗？

你好，能和您交流一下吗。

**Comment by hutchenky** [comment]
> Run the code and find that the RMSE and Score values of the test set are very large. Is the experimental result of the paper the result of the train set and the validation set?

您好，能跟您交流一下吗
